reportek/core/api/views/reporting.py

```diff
@@ -5,7 +5,7 @@
 import logging
 from base64 import b64encode
 from django.views import static
-from django.db.models import Q, F, Exists, OuterRef
```

Comment:
FilteredRelation not used in the end. :)

Reply:
Indeed, it was weak sauce :)